### PR TITLE
Moving Youtube player file and Ref anonymous func resolved

### PR DESCRIPTION
**Pull request by sarru1291**
**Description**

> What does this PR achieve? Is it a feature/hotfix/refactor? 
youtube player file moved to ``/sandbox/theatres/components/audio_player/players_by_type`` directory as suggested and using ref directly which was used using an anonymous function.

> What steps should the reviewer take to verify this PR resolves the issue?
Please review the new file structure of the youtube wrapper component.
Evidence:
Screenshot from 2019-04-20 21-55-01

@iisa

### CODE REVIEW

**Comment by Batey96** [comment]
@sarru1291  this PR refers to #110  which is meant to create a separate sandbox section for audio players..Its a work in progress because i have to manually look for every part of the codebase where each component,its styles and stories are being used ,then change the referenced
wip
 path

**Comment by sarru1291** [comment]
> @sarru1291 this PR refers to #110 which is meant to create a separate sandbox section for audio players..Its a work in progress because i have to manually look for every part of the codebase where each component,its styles and stories are being used ,then change the referenced

> path

**Comment by sarru1291** [comment]
> @sarru1291 this PR refers to #110 which is meant to create a separate sandbox section for audio players..Its a work in progress because i have to manually look for every part of the codebase where each component,its styles and stories are being used ,then change the referenced
> wip
> path

okay!! Have a look at some code changes as suggested.

**Comment by Batey96** [comment]
The work is completed..All referenced paths to this sandbox are updated and aware of its new location..

File: packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { Component} from 'react'
```
Comment (iisa) [inline packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js]: we use `;` here. is your eslinter working?

File: packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import PropTypes from 'prop-types'
+
+export default class YoutubePlayer extends Component {
```
Comment (iisa) [inline packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js]: please add JSDOC to describe component

File: packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  state={
```
Comment (iisa) [inline packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/youtube-player.js]: Please have constructor so you can use React's abilities

**Pull request closed without merging**